apm-agent-plugins/apm-servlet-plugin/src/main/java/co/elastic/apm/agent/servlet/ServletApiAdvice.java

```diff
@@ -143,6 +156,37 @@ public static void onEnterServletService(@Advice.Argument(0) ServletRequest serv
             servletTransactionHelper.fillRequestContext(transaction, request.getProtocol(), request.getMethod(), request.isSecure(),
                 request.getScheme(), request.getServerName(), request.getServerPort(), request.getRequestURI(), request.getQueryString(),
                 request.getRemoteAddr(), request.getHeader("Content-Type"));
+        } else if (transaction == null && servletRequest instanceof HttpServletRequest) {
+            final AbstractSpan<?> parent = tracer.getActive();
+            if (parent != null) {
+                final HttpServletRequest request = (HttpServletRequest) servletRequest;
+                DispatcherType dispatcherType = request.getDispatcherType();
+                boolean isAllowedType = false;
+                String spanAction = null, spanName = null;
+                if (dispatcherType == DispatcherType.FORWARD) {
+                    String pathInfo = request.getPathInfo();
+                    spanName = FORWARD + SPACE + request.getServletPath() + (pathInfo != null ? pathInfo : EMPTY);
+                    spanAction = FORWARD_SPAN_ACTION;
+                    isAllowedType = true;
+                } else if (dispatcherType == DispatcherType.INCLUDE) {
+                    Object pathInfo = request.getAttribute(RequestDispatcher.INCLUDE_PATH_INFO);
+                    spanName = INCLUDE + SPACE + request.getAttribute(RequestDispatcher.INCLUDE_SERVLET_PATH) + (pathInfo != null ? pathInfo : EMPTY);
+                    spanAction = INCLUDE_SPAN_ACTION;
+                    isAllowedType = true;
+                } else if (dispatcherType == DispatcherType.ERROR) {
+                    spanName = ERROR + SPACE + request.getAttribute(RequestDispatcher.FORWARD_SERVLET_PATH);
+                    spanAction = ERROR_SPAN_ACTION;
+                    isAllowedType = true;
```

Comment:
Please check if you already get the Exception object stored in the `RequestDispatcher.ERROR_EXCEPTION` attribute somewhere else. If not, let's create and send an error here with `captureException` through the parent span.